test/Microsoft.SqlTools.ServiceLayer.UnitTests/Profiler/ProfilerServiceTests.cs

```diff
@@ -3,66 +3,68 @@
+// // {
+// //     /// <summary>
+// //     /// Unit tests for ProfilerService
+// //     /// </summary>
+// //     public class ProfilerServiceTests
+// //     {   
+// //         /// <summary>
+// //         /// Test starting a profiling session and receiving event callback
+// //         /// </summary>
+// //         /// <returns></returns>
+// //         [Fact]
+// //         public async Task TestStartProfilingRequest()
+// //         {
+// //             string sessionId = null;
+// //             string testUri = "profiler_uri";
+// //             var requestContext = new Mock<RequestContext<StartProfilingResult>>();
+// //             requestContext.Setup(rc => rc.SendResult(It.IsAny<StartProfilingResult>()))
+// //                 .Returns<StartProfilingResult>((result) => 
+// //                 {
+// //                     // capture the session id for sending the stop message
+// //                     sessionId = result.SessionId;
+// //                     return Task.FromResult(0);
+// //                 });
 
-            var profilerService = new ProfilerService();
-            profilerService.SessionMonitor.AddSessionListener(sessionListener);
-            profilerService.ConnectionServiceInstance = TestObjects.GetTestConnectionService();
-            ConnectionInfo connectionInfo = TestObjects.GetTestConnectionInfo();
-            profilerService.ConnectionServiceInstance.OwnerToConnectionMap.Add(testUri, connectionInfo);
-            profilerService.XEventSessionFactory = new TestXEventSessionFactory();
+// //             var sessionListener = new TestSessionListener();
 
-            var requestParams = new StartProfilingParams();
-            requestParams.OwnerUri = testUri;
-            requestParams.TemplateName = "Standard";
+// //             var profilerService = new ProfilerService();
+// //             profilerService.SessionMonitor.AddSessionListener(sessionListener);
+// //             profilerService.ConnectionServiceInstance = TestObjects.GetTestConnectionService();
+// //             ConnectionInfo connectionInfo = TestObjects.GetTestConnectionInfo();
+// //             profilerService.ConnectionServiceInstance.OwnerToConnectionMap.Add(testUri, connectionInfo);
+// //             profilerService.XEventSessionFactory = new TestXEventSessionFactory();
 
-            await profilerService.HandleStartProfilingRequest(requestParams, requestContext.Object);
+// //             var requestParams = new StartProfilingParams();
+// //             requestParams.OwnerUri = testUri;
+// //             requestParams.TemplateName = "Standard";
 
-            // wait a bit for profile sessions to be polled
-            Thread.Sleep(500);
+// //             await profilerService.HandleStartProfilingRequest(requestParams, requestContext.Object);
 
-            requestContext.VerifyAll();
+// //             // wait a bit for profile sessions to be polled
+// //             Thread.Sleep(500);
```

Comment:
probably just bumping this up to something like 2000 will allow time for the threads to start on the slow CI build machines..through probably we need to add test hooks to allow the product to notify the test that the polling thread has initialized.  in any case, I'm fine with disabling until we pickup the profiling work again.